README.md

```diff
@@ -1,2354 +1,184 @@
-
-
-## Installation
-```
-pip install python-bitvavo-api
-```
-
-## Rate Limiting
-
-Bitvavo uses a weight based rate limiting system, with an allowed limit of 1000 per IP or API key each minute. Please inspect each endpoint in the [Bitvavo API documentation](https://docs.bitvavo.com/) to see the weight. Failure to respect the rate limit will result in an IP or API key ban.
-Since the remaining limit is returned in the header on each REST request, the remaining limit is tracked locally and can be requested through:
+<table>
+  <tr>
+    <td><a href="https://bitvavo.com"><img alt="Bitvavo" src="docs/assets/bitvavo-mark-square-blue.svg" width="100" title="Bitvavo Logo"></a></td>
+    <td><h1>Bitvavo SDK for Python</h1></td>
+  </tr>
+</table>
+
+Crypto starts with Bitvavo. 
+You use Bitvavo SDK for Python to buy, sell, and store over 200 digital assets on Bitvavo from inside your app. 
+
+To trade and execute your advanced trading strategies, Bitvavo SDK for Python is a wrapper that enables you to easily call every endpoint in [Bitvavo API](https://docs.bitvavo.com/).
+
+- [Prerequisites](#prerequisites) - what you need to start developing with Bitvavo SDK for Python
+- [Get started](#get-started) - rapidly create an app and start trading with Bitvavo
+- [About the SDK](#about-the-sdk) - general information about Bitvavo SDK for Python
+- [API reference](https://docs.bitvavo.com/) - information on the specifics of every parameter
+
+This page shows you how to use Bitvavo SDK for Python with WebSockets. 
+For REST, see the [REST readme](docs/rest.md).
+
+## Prerequisites
+
+To start programming with Bitvavo SDK for Python you need:
+
+- [Python3](https://www.python.org/downloads/) installed on your development environment
+
+  If you are working on macOS, ensure that you have installed SSH certificates:
+  ```terminal
+  open /Applications/Python\ 3.12/Install\ Certificates.command
+  open /Applications/Python\ 3.12/Update\ Shell\ Profile.command
+  ```
+- A Python app. Use your favorite IDE, or run from the command line
+- An [API key and secret](https://support.bitvavo.com/hc/en-us/articles/4405059841809) associated with your Bitvavo account
+
+  You control the actions your app can do using the rights you assign to the API key. 
+  Possible rights are:
+  + **View**: retrieve information about your balance, account, deposit and withdrawals
+  + **Trade**: place, update, view and cancel orders
+  + **Withdraw**: withdraw funds
+
+    Best practice is to not grant this privilege, withdrawals using the API do not require 2FA and e-mail confirmation.
+
+## Get started
+
+Want to quickly make a trading app? Here you go: 
+
+1. **Install Bitvavo SDK for Python**  
+
+    In your Python app, add [Bitvavo SDK for Python](https://github.com/bitvavo/python-bitvavo-api) from [pypi.org](https://pypi.org/project/python-bitvavo-api/):
+    ```terminal
+    python -m pip install python_bitvavo_api
+    ```
+1. **Create a simple Bitvavo implementation**
```

Comment:
:art: Should we not count upwards ourselves? 🥲 
```suggestion
2. **Create a simple Bitvavo implementation**
```

Reply:
Nope. Markdown numbering increments automatically, so you do 1. 1. 1. If you put 1. 2. 3. numbering, Markdown explicitly uses the number you write. Sooo, if you take out a step in the list and forget to update ALL the numbers afterwards, markdown rendering gives the wrong numbers. Best practice is to let Markdown do the work.